lua/nvim-training/tasks/yank_word.lua

```diff
@@ -1,67 +1,26 @@
 local utility = require("nvim-training.utility")
-local internal_config = require("nvim-training.internal_config")
-local YankTask = require("nvim-training.tasks.yank_task")
-local text_traversal = require("nvim-training.text_traversal")
-local YankWordTask = {}
-YankWordTask.__index = YankWordTask
+local YankTask = require("nvim-training.tasks.yank")
+local YankWord = YankTask:new()
+YankWord.__index = YankWord
 
-function YankWordTask:setup()
+function YankWord:new()
 	local base = YankTask:new()
-	setmetatable(base, { __index = YankWordTask })
+	setmetatable(base, { __index = YankWord })
 	self.autocmd = "TextYankPost"
 	self.target_text = ""
 	self.jump_distance = 2
 
 	local function _inner_update()
-		self.custom_lorem_ipsum = string.gsub(utility.lorem_ipsum_lines(), ",", "")
-		self.custom_lorem_ipsum = string.gsub(self.custom_lorem_ipsum, "%.", "")
-		utility.update_buffer_respecting_header(self.custom_lorem_ipsum)
+		utility.set_buffer_to_lorem_ipsum_and_place_cursor_randomly()
 
```

Comment:
This line is unexpected, and code should not surprise the readers. This line does.

Reply:
Hm, my comment is somewhat off, the comment is about the line where you set target text.